apis/eventsApi.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+const generateMockEvents = require('./mock/mockEventsApi');
+
+var events = [];
+
+events = generateMockEvents();
+
+// Get events with the given room and events in which the given time falls in between the event's start and end times
+function getEvents(room, time) {
+    var filteredEvents = events.filter(event => event.location == room && (time > event.startTime && event.endTime > time));
+    return filteredEvents;
+}
+
+// POST request body should be in JSON and include:
+// room: the room number to get events for
+// time: the date in milliseconds; any events happening at that date will be returned
+router.post('/getEvents', (req, res) => {
```

Comment:
This should be a GET, not a POST request – we're not changing any events by fetching them, nor are we sending data to be stored to the server. We're just GETting events.

Instead of sending stuff like room/time via the request body, just do it via the URL, like /getEvents/roomNumber/ – [this link](https://www.tutorialspoint.com/expressjs/expressjs_url_building.htm) explains the basics

Also, let's add some docs to the README about how to call the URL and what args to pass, as future devs shouldn't have to read through source code to figure this out

Also, consider making some defaults – for instance, if a time is not passed through, just use the current time.